package.json

```diff
@@ -110,6 +112,7 @@
     "react-dom": "18.3.1",
     "react-intl": "6.7.0",
     "react-router-dom": "6.26.2",
+    "rimraf": "6.0.1",
```

Comment:
Let's also update https://github.com/SonarSource/echoes-react/blob/main/package.json.md to explain why this is needed and used.